src/languageservice/services/validation/unused-anchors.ts

```diff
@@ -46,6 +51,17 @@ export class UnusedAnchorsValidator implements AdditionalValidator {
       }
     }
 
+    unIdentifiedAlias.forEach((node) => {
+      const nodeRange = node.range;
+      if (nodeRange) {
+        const startOffset = nodeRange[0];
+        const endOffset = nodeRange[1];
+        const range = Range.create(document.positionAt(startOffset), document.positionAt(endOffset));
+        const warningDiagnostic = Diagnostic.create(range, `Unresolved alias "${node}"`, DiagnosticSeverity.Hint, 0);
```

Comment:
Also maybe worth adding "it will be treated as a string instead" into the error message